nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -124,6 +124,11 @@ systemd.services.mysql.serviceConfig.ReadWritePaths = [ "/var/data" ];
      <varname>services.postfix.sslCACert</varname> was replaced by <varname>services.postfix.tlsTrustedAuthorities</varname> which now defaults to system certifcate authorities.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     If you're using a homebrew config to make steam run, you can now switch to the new steam module by enabling <varname>programs.steam.enable</varname> aswell as <varname>programs.steam.hardware</varname> for hardware-support.
```

Comment:
```suggestion
     If you're using a homebrew config to make steam run, you can now switch to the new steam module by enabling <varname>programs.steam.enable</varname> aswell as <varname>programs.steam.hardware</varname> for hardware-support.
```

homebrew config? I feel like that could get confused with homebrew the package manager https://brew.sh/

Reply:
The various documented workarounds to use steam have been converted to using a module. And direction that if you were using those workarounds documented in the manual you should migrate to using this module?

Maybe not exactly that, but that is what needs to be expressed in a release note.